sormas-backend/doc/UserDataAccess.md

```diff
@@ -1,62 +1,131 @@
+* all samples accessible for National Users
 * whoever created the sample or is assigned to it is allowed to access it
 * users see all samples of their cases
 * lab users see all samples of their laboratory
+* access by jurisdiction of the case or contact the sample belongs to
+* edit is possible only for samples of cases and/or contacts in the user's jurisdiction
+* view of sensitive data is possible only for samples that are part of the user's jurisdiction
 
-# Persons
+## Persons
 * all persons of the cases the user can access
 * all persons of the contacts the user can access
 * all persons of the events the user can access
+* edit is possible only only for persons that have their corresponding contact, case or event in the users jurisdiction
+* view of personal and sensitive data is possible only for persons that have their corresponding contact, case or event in the users jurisdiction
 
-# WeeklyReports
+## WeeklyReports
 * only users that are permitted to see weekly reports at all
 * whoever created the weekly report
 * national users see all weekly reports in the database
 * supervisors see all weekly reports from facilities in their region
 * officers see all weekly reports of their informants
 
-# WeeklyReportEntries
+## WeeklyReportEntries
 * users can see all weekly report entries associated with weekly reports they can access
+
+# User notifications
+
+## Case changed notification
+* Types: email, sms
+* To: all responsible supervisors within the case region
+* When: case classification changed or disease of unspecified VHF changed
+
+## Case investigation status done notification
+* Types: email, sms
+* To: all responsible supervisors within the case region
+* When: case investigation status gets updated to done
+
+## Pathogen test result notification
+* Types: email, sms
+* To: all responsible supervisors within the region of corresponding case or contact
+* When: 
+    * create of pathogen test with test result different from PENDING
+    * update of pathogen test result from PENDING to any other value
+    
+## Contact follow-up task done notification
+* Types: email, sms
+* To: all responsible supervisors within the region of the task
+* When: task of type contact follow-up set to status done
+
+## New and pending task notification
+* Types: email, sms
+* To: [assigned user](#task-generation-and-user-assignment) of each new or pending task
+* When: every 10 minutes 
+
+## Visit changed notification
+* Types: email, sms
+* To: surveillance and contact supervisor of visits contacts region
+* When: visit having contacts becomes symptomatic
+
+# Task generation and user assignment
+
+## Submit weekly reports
+* generated every hour if WEEKLY_REPORT_GENERATION feature enabled for each hospital informant
+
+## Contact follow-up
+* generated every hour if CONTACT_FOLLOW_UP feature enabled 
+* assigned user becomes the contact officer if exists, if not a random officer of the contact district
```

Comment:
I don't really get what this is supposed to mean

Reply:
it is a description of the logic how the assignee is choosen in the ContactFacadeEjb.generateContactFollowUpTasks method - thought it's relevant to whom a task get's assigned and that's why included it here
should i reformulate it ?

Reply:
Ah, okay, I see. I'd rephrase it as follows:

"the task assignee is set to the contact officer if specified, otherwise to a random officer in the contact district"

And basically the same concept for the open discussion point below.